presto-server-rpm/src/test/java/io/prestosql/server/rpm/ServerIT.java

```diff
@@ -13,27 +13,66 @@
  */
 package io.prestosql.server.rpm;
 
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.GenericContainer;
 import org.testng.annotations.Parameters;
 import org.testng.annotations.Test;
 
 import java.sql.Connection;
 import java.sql.ResultSet;
+import java.sql.SQLException;
 import java.sql.Statement;
 import java.util.HashSet;
 import java.util.Set;
 
+import static java.lang.String.format;
 import static java.sql.DriverManager.getConnection;
 import static java.util.Arrays.asList;
+import static org.testcontainers.containers.wait.strategy.Wait.forLogMessage;
 import static org.testng.Assert.assertEquals;
 
 public class ServerIT
 {
-    @Parameters("presto.port")
+    @Parameters({"target", "rpm"})
```

Comment:
you can pass `target/server.rpm` as single param and bind file in the docker

Reply:
Done